### PR TITLE
Add memory check warning before dev.up

### PR DESCRIPTION
**Pull request by macdiesel**
Print out a low memory warning before starting docker containers with make.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -108,3 +108,6 @@ create-course-studio: ## Generates a course on studio using the configurations i
 
 create-course-ecommerce: ## Generates a course on ecommerce using the configurations in test-course-ecommerce.json
 	./course-generator/create-course-ecommerce.sh
+
+check-memory:
+	@if [ `docker info --format '{{json .}}' | python -c "import sys, json; print json.load(sys.stdin)['MemTotal']"` -le 2147483648 ]; then echo "\033[0;31mWarning, System Memory is set too low!!! Increase Docker memory to be at least 2 Gigs\033[0m"; fi || exit 0
```
Comment (clintonb) [inline Makefile]: Add a newline.

**Review by clintonb — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (macdiesel) [reply Makefile]: done

File: Makefile [hunk 2]
```diff
@@ -108,3 +108,7 @@ create-course-studio: ## Generates a course on studio using the configurations i
...
```
Comment (staubina) [inline Makefile]: Is 2 GB enough?

Comment (macdiesel) [reply Makefile]: We'll start here and increment as necessary.

**Pull request merged**